### PR TITLE
Added PositionUtils class in the sdk-client

### PR DESCRIPTION
**Pull request by piotrwitek**
Added PositionUtils class in the sdk-client so the front-end can use that to calculate and show extra position info in the UI

### CODE REVIEW

File: sdk/sdk-client/src/utils/PositionUtils.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+export class PositionUtils {
+  static getLTV({
+    collateralTokenAmount,
+    debtTokenAmount,
+    collateralPrice,
+    debtPrice,
+  }: {
+    collateralTokenAmount: TokenAmount
+    debtTokenAmount: TokenAmount
+    collateralPrice: string
```
Comment (robercano) [inline sdk/sdk-client/src/utils/PositionUtils.ts:13]: Should we pass it as `Price`?

File: sdk/sdk-client/src/utils/PositionUtils.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    debtPrice: string
+  }): Percentage {
+    // Determine the Collateral Value:
+    const collValue = new BigNumber(collateralTokenAmount.amount).times(collateralPrice)
```
Comment (robercano) [inline sdk/sdk-client/src/utils/PositionUtils.ts:17]: I'd prefer to modify `TokenAmount.multiply` to accept a `Price` instead (for non-USD prices) or a Utils to handle this multiplicaiton. If this is needed urgently, at least indicate the quote currency for the price. i.e.: `collateralPriceInUSD`

File: sdk/sdk-client/src/utils/PositionUtils.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  static getLiquidationPrice({
+    position,
+    liquidationThreshold,
+    debtPrice,
+  }: {
+    position: Position
+    // TODO: passed as param because it is not defined in Position
+    liquidationThreshold: Percentage
```
Comment (robercano) [inline sdk/sdk-client/src/utils/PositionUtils.ts:36]: You can get it from Pool, but I think you are not getting Pool from the SDK at this moment. If you don't mind, update the TODO to suggest that we pass Pool in the future

File: sdk/sdk-client/src/utils/PositionUtils.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    collateralPrice: string
```
Comment (robercano) [inline sdk/sdk-client/src/utils/PositionUtils.ts:37]: Same comment for these prices as for the other function

File: sdk/sdk-client/src/utils/PositionUtils.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    // If the collateral value is 0, return 0.
+    if (collValue.isZero()) {
+      return Percentage.createFrom({ percentage: 0 })
+    }
+    // Divide the borrowed value by the collateral value and multiply by 100 to get the percentage LTV.
+    const ltvRatio = debtValue.div(collValue).times(100).toNumber()
+    return Percentage.createFrom({ percentage: ltvRatio })
+  }
+
+  static getLiquidationPrice({
```
Comment (robercano) [inline sdk/sdk-client/src/utils/PositionUtils.ts:29]: `getLiquidationPriceInUSD`

File: sdk/sdk-client/tests/PositonUtils.spec.ts [hunk 6]
```diff
@@ -0,0 +1,131 @@
+import { Percentage, TokenAmount, type Position } from '@summerfi/sdk-common/common'
+
+import { PositionUtils } from '../src/utils/PositionUtils'
+import { WETH, DAI } from './TestUtils'
+
+describe.only('PositionUtils', () => {
```
Comment (robercano) [inline sdk/sdk-client/tests/PositonUtils.spec.ts:6]: Remove `only`

**Review by robercano — APPROVED** [review]
Looks good to me! Approving but please have a look at the suggestions

File: sdk/sdk-client/src/utils/PositionUtils.ts (hunk 1 above)
Comment (piotrwitek) [reply sdk/sdk-client/src/utils/PositionUtils.ts:13]: done

File: sdk/sdk-client/src/utils/PositionUtils.ts (hunk 3 above)
Comment (piotrwitek) [reply sdk/sdk-client/src/utils/PositionUtils.ts:36]: done

File: sdk/sdk-client/src/utils/PositionUtils.ts (hunk 4 above)
Comment (piotrwitek) [reply sdk/sdk-client/src/utils/PositionUtils.ts:37]: done

File: sdk/sdk-client/src/utils/PositionUtils.ts (hunk 2 above)
Comment (piotrwitek) [reply sdk/sdk-client/src/utils/PositionUtils.ts:17]: done

**Review by robercano — APPROVED** [review]
Looks great!

**Review by robercano — APPROVED** [review]

**Pull request merged**